### PR TITLE
[blog] add related posts

### LINKED ISSUE #312: [posts] add related_posts
- [x] add id connections
- [x] add section under/above post

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #312

### CODE REVIEW

**Comment by TomasVotruba** [comment]
### Before

image


### After

image

**Comment by TomasVotruba** [comment]
### Before 

image


### After

image

or no related

image

**Comment by TomasVotruba** [comment]
Or no headline (there quite a lot of them) and right to the point?

image

**Comment by TomasVotruba** [comment]
I've added related posts section under every post, that has some related posts.

So what do you think - what would you improve?

**Comment by TomasVotruba** [comment]
@enumag @f3l1x @paveljanda @jakubkratina Could I ask you for short review?

File: source/_layouts/post.latte [hunk 1]
```diff
@@ -31,13 +31,16 @@
...
+
             <div class="intermezzo-smaller">
                 <div class="container">
-                    <h2>Share your View in the Comments</h2>
+                    {include relatedPosts, 'post' => $post}
                 </div>
             </div>
 
             <div class="container">
+                {*<h2>Share your View in the Comments</h2>*}
```
Comment (grogy) [inline source/_layouts/post.latte]: This you can remove

**Comment by TomasVotruba** [comment]
Thank you guys, let's deploy this

**Comment by enumag** [comment]
@TomasVotruba The feature is great but it currently shows link to articles that were not released yet.

https://pehapkari.cz/blog/2017/08/28/arachne-verifier-request-validator-for-nette-application/

This could be fixed though in my opinon.

**Comment by TomasVotruba** [comment]
@enumag I agree :+1: 

I was hoping you'd write this :). Could you send PR to Statie to show only unreleased posts?

**Comment by enumag** [comment]
@TomasVotruba Not any time soon. :-(

**Comment by TomasVotruba** [comment]
@enumag No problem, any time late will do :+1: . This is not security issue

**Comment by f3l1x** [comment]
PR is good by the way ;-)

**Pull request merged**